app/src/main/java/org/fossasia/openevent/events/TracksDownloadEvent.java

```diff
@@ -0,0 +1,20 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class TracksDownloadEvent {
```

Comment:
There is repeated code. Can this be refactored?  TracksDownloadEvent, VersionDownloadEvent there is only difference in the name of the event.